### ISSUE TITLE
Type() is not clearing the input before typing in React component (serenity3.6.7)

### ISSUE THREAD

**Issue by mrpet88** [serenity-bdd/serenity-core#3078]
It seems that type() method is not working properly with latest version of serenity, in React components. Since I can not provide much details from my company's project, I created two repositories, where the issue is reproducible. 

**Problem: Typing on an React input component that already has value, is not clearing the input, but instead it types next to the previous value.** 

**Info:** 
- Works properly on old serenity 3.0.5 with selenium 3.x.x
- Does not work with new serenity with selenium 4.x.x
- Checked on firefox, still same behaviour
- Checked on different driver version, still same behaviour
- Type() works properly with serenity 3.6.7 on `non-react` component

**Repositories to reproduce:**
- https://github.com/mrpet88/serenity-cucumber-selenium3
- https://github.com/mrpet88/clear-issue-new-serenity

**Image:**
- ![clear-issue-serenity](https://user-images.githubusercontent.com/40420772/223223034-dbe88804-8746-4424-a6dc-614f585b30bf.png)

P.S. With pure selenium 3.x.x and 4.x.x, `clear()` is not working properly as well, but since clear seems to be working properly with serenity 3.0.5, I thought is a good idea to post this issue here, besides selenium.

**Comment by mrpet88** [comment]
One guess is that this is why it works properly on serenity 3.0.5:

```
if (!isMobileDriver()) {
            ClearContents.ofElement(getElement());
        }
        getElement().clear();
      
```
```
  public static void ofElement(WebElement element) {
        element.sendKeys(COMMAND_A, DELETE);
        element.sendKeys(CONTROL_A, DELETE);
    }
```

**Comment by wakaleo** [comment]
It's not the Serenity version, it's related to the underlying Selenium verion. I believe this is related to inconsistant behaviour of the clear method on different browsers, devices and OSs. The Control-A/Delete hack stopped working for some contexts in more recent versions of Selenium. I don't have a fix.

**Comment by mrpet88** [comment]
Yeah I also sent to selenium about this, unfortunatelly I don't think it will be handled there as well. Thanks @wakaleo

**Comment by TimB0** [comment]
ClearContents.ofElement(getElement());  is currently working for me in Serenity 3.7.1
